cfgov/cfgov/settings/local.py

```diff
@@ -23,8 +23,16 @@
 INSTALLED_APPS += (
     "sslserver",
     "wagtail.contrib.styleguide",
+    "django_elasticsearch_dsl",
 )
 
+ELASTICSEARCH_DSL={
+    'default': {
+        'hosts': 'http://localhost:9200'
```

Comment:
Could we make this dependent on an environment variable and add that variable to `.env_sample`? I have to change it to get it to work locally in Docker.

Reply:
Added ES7_HOST variable to `.env_sample` and used that ES7 variable in `ELASTICSEARCH_DSL`